core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1025,6 +1034,11 @@ public void requestTrailers(HttpHeaders requestTrailers) {
         updateFlags(RequestLogProperty.REQUEST_TRAILERS);
     }
 
+    @Override
+    public int currentAttempt() {
+        return currentAttempt;
```

Comment:
I was wondering if we could infer the current attempt from ` parent.children().indexOf(this) + 1`.

Reply:
That's a good idea. 👍 I haven't thought of it in that way. Let me fix that. Thanks!